sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -435,16 +436,16 @@ def query_entities(
             filter,  # type: str  # pylint: disable = W0622
             **kwargs
     ):
-        # type: (...) -> ItemPaged[TableEntity]
+        # type: (...) -> ItemPaged[azure.data.tables.TableEntity]
```

Comment:
In mypy typing, since youy've already imported `TableEntity` up above, you don't need the full path. Just to put everything in one place:

mypy: `ItemPaged[TableEntity]`
sphinx: `:rtype: ~azure.core.paging.ItemPaged[~azure.data.tables.TableEntity]`